src/pages/publication/publication_page/PublicationPageAuthors.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+import { Contributor } from '../../../types/contributor.types';
+import NormalText from '../../../components/NormalText';
+import styled from 'styled-components';
+
+const StyledAuthors = styled(NormalText)``;
+
+const StyledAuthor = styled.span`
+  margin-right: 1rem;
+`;
+
+interface PublicationPageProps {
+  authors: Contributor[];
+}
+
+const PublicationPageAuthors: FC<PublicationPageProps> = ({ authors }) => {
+  return (
+    <StyledAuthors>
+      {authors.map((author, index) => {
```

Comment:
tviler på at det gjør noe i dette tilfellet, men til senere: https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Contibutor har ingen unike verdier pr nå, og det er ikke sikkert vi kan få det. Kanskje vi burde tildele id'er ved henting, slik at vi unngår sorterings-problemer beskrevet i artikkelen ?  @elkebab

Reply:
nesj, har vi ingen god id lever jeg fint med dette :)